src/sentry/api/bases/integration.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import six
+
+from rest_framework.response import Response
+
+from sentry.integrations.exceptions import ApiHostError
+from sentry.utils.sdk import capture_exception
+
+from .organization import OrganizationEndpoint, OrganizationPermission
+
+
+class IntegrationEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission, )
+
+    def handle_exception(self, request, exc):
+        if isinstance(exc, ApiHostError) or exc.code == 503:
+            import sys
+            import traceback
```

Comment:
You can import these at the top of the module. They won't ever create cyclic imports.